### PR TITLE
MRG: Add news [ci skip]

### LINKED ISSUE #3703: ENH?: News / updates
It seems like we should have a little news/updates section of one-liners on the website, including things like:
1. Release notifications
2. Upcoming MNE-Python workshops
3. Upcoming coding sprints

If people agree I can put some old ones (last couple of release dates), and we can add to it as announcement-worthy things come up.

### PR DESCRIPTION
**Pull request by Eric89GXL**
Closes #3703.

### CODE REVIEW

File: doc/index.rst [hunk 1]
```diff
@@ -83,41 +88,37 @@
...
+    - October 1\ :sup:`st`
+        MNE-Python poster at BioMag 2016, Seoul, South Korea.
+    - September 29\ :sup:`th`
+        `BRAIN grant <https://news.brown.edu/articles/2016/09/brain>`_
+        funded to use MNE to better understand multi-modal recordings.
+    - September 27\ :sup:`th`
+        `Version 0.13 <https://pypi.python.org/pypi/mne/0.13>`_ released
+    - March 10\ :sup:`th`
+        `Google Summer of Code 2016 project <https://summerofcode.withgoogle.com/projects/#5290867271139328>`_
+        funded to improve :ref:`api_decoding` module.
```
Comment (dengemann) [inline doc/index.rst]: how will it work, can we just add the news here?
I'd then add a few.

**Comment by Eric89GXL** [comment]
Yeah feel free to commit to my branch now if you have access otherwise PR

**Comment by dengemann** [comment]
> Yeah feel free to commit to my branch now if you have access otherwise PR

I mean in general, the workflow will be to update this section of that file?

**Comment by Eric89GXL** [comment]
Yes

**Comment by agramfort** [comment]
I'd rather have an mne_news twitter feed that we embed in the doc.

**Comment by Eric89GXL** [comment]
Then one person is the gatekeeper, right? I assume you're volunteering? :)

**Comment by agramfort** [comment]
we should give the pass to a handful of people so we can share the work/responsability.

**Comment by Eric89GXL** [comment]
Should we get rid of the commit feed / take that over? It's not too useful as is

**Comment by agramfort** [comment]
yes that would be fine with me

**Comment by teonbrooks** [comment]
I would be up for gatekeeping the twitter account. maybe after we get it from littering with merges, I will follow it myself ;)

**Comment by jasmainak** [comment]
go go @teonbrooks ! :)

**Comment by Eric89GXL** [comment]
If we're going the twitter route, this one is ready for merge

https://3065-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/index.html

**Comment by teonbrooks** [comment]
so the twitter feed will be releases, events, and talks?

**Comment by agramfort** [comment]
I would keep a "getting help" section on the right with links to mailing
list and issue tracker

otherwise fine with me

**Comment by Eric89GXL** [comment]
@agramfort done

https://3092-1301584-gh.circle-artifacts.com/0/home/ubuntu/mne-python/doc/_build/html/index.html

**Comment by Eric89GXL** [comment]
@agramfort this one is good to go

**Comment by agramfort** [comment]
thx @Eric89GXL

**Pull request merged**